verse-pbc/plur
# Show an error message if joining a group fails

## Issues covered
https://github.com/verse-pbc/issues/issues/108

## Description
Shows an error message if the user can't join the group

## How to test
1. As a group admin, navigate to a group
2. Tap the 🫂+ button to create an invitation link
3. Copy the link
4. Either switch accounts on the same device or use another device with another account
5. Paste the link into the browser
6. Modify the link by deleting a character from the code
7. Tap Return
8. Tap Continue to open Plur
9. Observe the error toast that's displayed at the bottom of the screen

## Screenshots/Video
Post screenshots or video showing your changes, ideally showing how the app worked before and after these changes. Delete this section if this PR contains no visual changes.

### Before

https://github.com/user-attachments/assets/34e03ec7-29e6-4c30-afd2-e72b606290d7

### After

https://github.com/user-attachments/assets/d296e0be-eb47-441f-9df4-04296fb2e52f

## To do
- [x] if time, refactor a bit to make this look nicer
